app/views/bands/edit.html.erb

```diff
@@ -0,0 +1,23 @@
+<h1>Update <%= @band.name %> </h1>
+
+<%= form_for (@band) do |f| %>
+  <div class="field">
+    <%= f.label :name, "Band Name" %>
+    <%= f.text_field :name %>
+  </div>
+  
+  <div class="field">
+    <%= f.label :biography %>
+    <%= f.text_area :biography %>
+  </div>
+
+  <div>
+    <h3>(Optional): Upload Band Profile Image</h3>
+    <%= f.file_field :image%>
+  </div>
+
+  <%= f.submit "Make Changes"%>
+<%end%>
+</br>
+</br>
+<%=button_to "Delete Band", @band, :method => :delete, data: { confirm: "You sure?" } %>
```

Comment:
im not a huge fan of the prompt, but ill allow it. however `:method => :delete` uses old syntax. Change to 
`method: :delete`